src/OrbitGl/CallTreeView.cpp

```diff
@@ -152,6 +182,11 @@ static void AddUnwindErrorToTopDownThread(
       unwind_errors_node, frame, function_name, module_path, module_build_id.value_or(""));
   function_node->IncreaseSampleCount(callstack_sample_count);
   function_node->AddExclusiveCallstackEvents(callstack_events);
```

Comment:
If I remember correctly, now this should be removed otherwise you are calling `AddExclusiveCallstackEvents` with the same `callstack_events` twice. Please make sure that the "Select these callstacks" context menu item works correctly after the change.

Reply:
That means that if you use "Select these callstacks" on the `CallTreeUnwindErrorType` it won't work. The idea is that the `callstack_events` are added to the node that is the furthest down the tree (hence, exclusive), and then the algorithm behind "Select these callstacks" will go over the descendants of the selected node(s) and collect the `CallstackEvent`s.

Reply:
We might want to discuss this offline, because with the `CallTreeUnwindErrorType` under the functions "exclusive" is weird now. An option is just to move the types above the functions similarly to the bottom-up view and call it a day.

Reply:
As discussed offline. Now TopDown looks like:
```
Thread
|-> [unwind errors]
       |-> error type1
       |     |-> FunctionA
       |     |-> FunctionB
       |-> error type 2
```